mu4/domain/notation/inotation.h

```diff
@@ -66,6 +67,9 @@ class INotation : MODULE_EXPORT_INTERFACE
     // playback (midi)
     virtual INotationPlayback* playback() const = 0;
 
+    // elements
+    virtual INotationElementsRepository* elementsRepository() const = 0;
```

Comment:
I didn't like the use of the `repository` term in the inspector, and now I don't like it either. The term `repository` is about accessing a repository/storage (database, file system, in-memory storage). Usually, it is at on the data layer level. And here we are talking about access to the model. Maybe just `INotationElements`.

Reply:
mm, you know, I wasn't sure about "repository" term here, that was just a guess.
And, actually, now I think that it was a mistake to use that term in inspector module, because it's not seem like a "low-level" service, you know